fedimint-bitcoind/src/bitcoincore_rpc.rs

```diff
@@ -18,7 +18,13 @@ pub fn make_bitcoind_rpc(cfg: &BitcoindRpcCfg, task_handle: TaskHandle) -> Resul
         Auth::UserPass(cfg.btc_rpc_user.clone(), cfg.btc_rpc_pass.clone()),
     )
     .map_err(anyhow::Error::from)?;
-    let retry_client = RetryClient::new(Client(ErrorReporting::new(bitcoind_client)), task_handle);
+    let retry_client = RetryClient::new(
+        Client(ErrorReporting::new(
+            cfg.btc_rpc_address.clone(),
```

Comment:
Will this field include the password in the future? I know lots of software allows passing a single `user:pass@host:port` string. Is it OK to log a password like that?

Reply:
If RPC were somehow credentials leaked that could be pretty disastrous

Reply:
I'm pretty sure that we do not support including credentials like that rn and would just ignore them. That being said, this might change in the future and maybe we should impl a custom rendering fn to censor user+password.

Reply:
We don't support credentials like that now, just wanted to mention that in case we do.

OK well I'm going to approve.